### ISSUE TITLE
OS X Nightly build 2017-11-05 crashes on startup

### ISSUE THREAD

**Issue by ehlers** [GNS3/gns3-gui#2325]
OS X v10.11.6 (El Capitan)

GNS3 nightly build 2.1.0dev9 from 2017-11-05 crashes immediately during the start.
The nightly build 2.1.0dev9 from 2017-11-04 runs fine.

```
/Users/behlers# /Applications/GNS3.app/Contents/MacOS/gns3
Fatal Python error: Py_Initialize: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007fff736be000 (most recent call first):
Abort trap: 6
/Users/behlers#
```

**Comment by ehlers** [comment]
I have an idea, what happened:
The new version 5.1 of cx_Freeze has been released on the 4th of November. Some changes, maybe https://github.com/anthony-tuininga/cx_Freeze/pull/286 has broken it.

**Comment by ehlers** [comment]
Just tried the nightly build 2.1.0dev9 from 2017-11-06, same issue.

**Comment by ziajka** [comment]
Hi @ehlers,
Thanks for letting as know. I've frozen version of cx_Freeze. Let's wait 24h for the new nightly. 
Cheers,
Dominik

**Comment by ehlers** [comment]
Just tested with nightly build 2.1.0dev10 from 2017-11-08, now GNS3 works again. Issue seems resolved.

**Comment by ziajka** [comment]
Great, thanks @ehlers!

**Comment by ehlers** [comment]
To get this fixed I've opened issue https://github.com/anthony-tuininga/cx_Freeze/issues/324

**Comment by ziajka** [comment]
Thanks @ehlers

**Comment by ehlers** [comment]
@ziajka The bugs of cx_Freeze v5.1 seems to be fixed now, the version 5.1.1 was just released. My simple test programs are working fine. You might give it a try with GNS3.

**Comment by grossmj** [comment]
We will try soon, thanks for the feedback! :+1:
